app/assets/stylesheets/application.css

```diff
@@ -115,3 +115,11 @@ a:hover {
 a:active {
   color: blue;
 }
+
+.pet_link {
```

Comment:
these links added extra styling to the links and allow for future styling as well as helping with capybara testing